scripts/slideshow.js

```diff
@@ -25,11 +26,11 @@
         // jQuery shortcut
         var $ = jQuery;
         // if the height is not set then wait a bit to initialize the slideshows
-        if (!$('.slideshow .slideshow-container .slides').height()) {
+        if (!$('[doc-slideshow] .slides').height()) {

```

Comment:
.slides is just css class, maybe better to use that u="slides" ?

Reply:
Done.